sites/kit.svelte.dev/src/lib/search/SearchBox.svelte

```diff
@@ -238,11 +239,16 @@
 	}
 
 	input:focus-visible {
+		--c: white;
 		background: var(--second);
-		color: white;
+		color: var(--c);
 		outline: none;
 	}
 
+	input::placeholder {
+		color: var(--c);
```

Comment:
I think this should be faded, in the way that placeholders are by default on a white background — something like `rgba(255, 255, 255, 0.5)` (but only when `:focus-visible`, obviously)